modules/display.py

```diff
@@ -184,7 +184,7 @@ def message(self, message, showConfig=True):
 
         url = 'caption:'
         if helper.getDeviceIp() is not None and showConfig:
-            url = 'caption:Configuration available at http://%s:7777' % helper.getDeviceIp()
+            url = 'caption:Configuration available at http://%s' % helper.getDeviceIp()
```

Comment:
Going to have to fix this later so it pulls that info from the settings 😄

Reply:
I was looking at this, and thought it might be a simple fix (copying the server port from the code at line 175).  But that code crashes when I use it here.  I get a "server" is not defined when I try to use server.get_server_port().   Seems there may be a bug at 175 as well.

Reply:
I've been thinking about this too, server will fail since it's not exposed to this namespace, an option is to have the server expose a hook that the helper registers to. But like I mentioned, this doesn't have to be solved at this time since the default is 80 and anyone changing will hopefully also know what they're doing 😄

Reply:
What about frame.py?  It parses the cmd line args and sets the port to start the server.  So, instead of asking the server what port it is at, we collect the information at the start of the program and make it available with all the other base configuration variables?